server/controller/src/main/java/ai/starwhale/mlops/api/JobApi.java

```diff
@@ -266,4 +267,18 @@ ResponseEntity<ResponseMessage<String>> removeJob(
     ResponseEntity<ResponseMessage<String>> recoverJob(
             @Valid @PathVariable("projectUrl") String projectUrl,
             @Valid @PathVariable("jobUrl") String jobUrl);
+
+    @Operation(summary = "Create a new model serving job")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "ok")})
+    @PostMapping(value = "/project/{projectUrl}/serving")
+    @PreAuthorize("hasAnyRole('OWNER', 'MAINTAINER')")
+    ResponseEntity<ResponseMessage<String>> createModelServing(
```

Comment:
could we list or kill it

Reply:
Yes, we should support these actions, I will implement them in the later PR. (The PR will be too big if it contains the whole features)